.gitignore

```diff
@@ -1,3 +1,6 @@
+# Creds file
+creds.cfg
```

Comment:
can we add in a sample creds file so folks know what to create?

Reply:
creds.cfg already exists just adding the ignore here.

Reply:
Renamed it to creds.cfg.sample